src/rest/server/auth.go

```diff
@@ -38,10 +38,10 @@ type UserCredential struct {
 }
 type UserAuth map[string]bool
 
-var ClientAuth = UserAuth{"password": false, "cert": false, "jwt": false, "cliuser": false}
-
-
 func (i UserAuth) String() string {
+	if i["none"] {
```

Comment:
I don't see `i["none"]` getting set anywhere other than in the `Set` method, and that too, only if the input string is `none` or empty. Can you confirm that this doesn't cause any issues for the server? My recollection is that Go will panic if the key doesn't exist in the map. Typically, this is resolved by something like the following.

```if val, ok := i[key]; ok && cond(val) {```

Reply:
No, if the value is not in the map, it returns false. Setting a value in a nil map will cause a panic, but we don't have a nil map here and we are not setting a value here. From the go docs:

`If the requested key doesn't exist, we get the value type's zero value. `